app/models/shipit/user.rb

```diff
@@ -123,5 +123,28 @@ def identify_renamed_user!
     rescue Octokit::NotFound
       false
     end
+
+    def org_email?(email_address)
+      return false if email_address.blank?
+
+      org_domains = Shipit.preferred_org_emails
+      return true if org_domains.blank?
```

Comment:
This check should be first, this way if the option is disabled, we won't fetch private emails even if the public one is blank.

Reply:
Also since it return true when the option is disabled, the name is kinda weird, but no huge deal.

Reply:
We want to fetch private emails when the public one is blank, regardless of whether the org_emails filter is set or not.

Reply:
I think if the `org_emails` filter is missing we can assume the operator doesn't care about this, and we can leave it blank.

Reply:
Alrighty. I'll adjust (and update the source issue).